packages/backend/src/models/table-collaborators.ts

```diff
@@ -66,6 +69,16 @@ class TableCollaborator extends Base {
       permissionLevels.indexOf(collaborator.role) <
         permissionLevels.indexOf(role)
     ) {
+      if ($) {
+        throw new StepError(
+          'You do not have access to this tile',
+          `Please ensure that you are ${
+            role === 'viewer' ? 'a' : 'an'
+          } ${role} of this tile.`,
+          $.step.position,
+          $.app.name,
+        )
+      }
```

Comment:
dumb qns but actly can i check would you ever run into the situation where the `role` is a viewer? isit when you are not a collaborator or smth?